src/cli/README.md

```diff
@@ -807,6 +808,25 @@ Delete all stored settings, and signal a platform reset.
 > factoryreset
 ```
 
+### fem
+
+Get the external FEM parameters.
+
+```bash
+> fem
+LNA gain 11 dBm
+Done
+```
```

Comment:
```suggestion
### fem lnagain

Get LNA gain in dBm of the external FEM.

```bash
> fem lnagain
11
Done
```
```
```suggestion
### fem

Get the external FEM parameters.

```bash
> fem
LNA gain 11 dBm
Done
```
```

Reply:
For this, I would like to use `fem` command to retrieve all the fem parameters so no subcommand is needed. How do you think?

Reply:
Is it more consistent to have getter and setter the same pattern? I treat the CLI another protocol to control OpenThread. As you mentioned, the unit is already documented in API and README of the CLI. `LNA gain 11 dBm` uses more bytes and is harder to parse than `11` for scripts.